### DISCUSSION TITLE
elfmz, I didn't thank you enough!

### DISCUSSION THREAD

**Discussion by singalen** [elfmz/far2l#927] [General]
It's so amazing to work in a better operating system with the best file manager.
The amount of work you put in is massive, it qualifies for a part-time job.
I'm immensely thankful.

**Comment by paksv** [comment]
I just can't express enough my gratitude for the thing you've done. Thank you, thank you, thank you!

**Comment by unxed** [comment]
I also want to express my gratitude. far2l is a tool that I use every day and can't imagine my work without it. At the same time, I want to note that the absence of elfmz from the 24th of February caused concern. @elfmz glad to see you on github again! I want to hope that you are ok, and that the catastrophic events that are taking place have not touched you.

**Comment by ha3flt** [comment]
@elfmz, thank you, and let me tell you the same as others did above. My whole professional life (30+ years) has been made million times easier as it could be by the NC, VC, later FAR on Windows, and now far2l in Linux even if I use it mostly at home - that's important, too. I could make its almost all shortcuts working (even in a terminal) by moving my distro's less important global shortcuts out of the way (adding a plus Shift to them, or so).

Most people still don't understand that working with file systems needs a similar IDE as it is widely accepted for programming languages, and also did not experience the difference in speed of completing tasks by a tool being so ergonomic and well thought out, and reach of functions. No other software gives these integrated and available by simple keyboard shortcuts, and this non-proportional character-based UI is somehow just much better than the "graphical" ones.

I love the customizable panel columns (I always make FAR and far2l to display Group, Owner, and Attributes of files and directories in every line), Find file's functionality, History panels, being able to look inside a zip-compressed file having any extension (e.g. an .xlsx file), placing the path of the panels to wherever I need it, also the customisable plugins for highlighting, working with SFTP using the same panels as the remote files and folders were local, decompressing files as it was a simple copy between folders, etc. etc. etc.

Hopefully I will get to the point I can help improving far2l by myself, too. I had one small pull request already accepted but that's not too much yet. :-)
